docs/validators/advanced-setups.mdx

```diff
@@ -0,0 +1,102 @@
+---
+description: Using Connect with Advanced Validator Infrastructure
+title: Advanced Setups
+icon: plug
+---
+
+## Using Remote Signers <Icon icon="key-skeleton-left-right" />
+
+Remote signers can be used with Connect, however only certain versions are compatible.
+
+### Horcrux
+
+#### Required Version
+
+v3.3.0+
+
+#### With `Horcrux-Proxy`
+
+If you are using horcrux-proxy be sure to use v1.0.0+ so that the maxReadSize can be configured in accordance with larger vote-extension payloads
```

Comment:
```suggestion
If you are using `horcrux-proxy` be sure to use v1.0.0+ so that the maxReadSize can be configured in accordance with larger vote-extension payloads
```